roles/provision-keycloak-apb/tasks/provision-keycloak.yml

```diff
@@ -223,16 +223,27 @@
 - set_fact: installation="{{ installation_text.stdout | from_json }}"
 - set_fact: installationbearer="{{ installation_bearer.content | from_json }}"
 
+- k8s_v1_secret:
+    name: keycloak
+    namespace: '{{ namespace }}'
+    labels:
+      name: keycloak
```

Comment:
@philbrookes this seems redundant. Does the mobile-cli need the name to be in a label too?

Reply:
I think this can be removed.